NOTICE-3RD-PARTY-CONTENT.md

```diff
@@ -40,21 +40,17 @@
 ## Workflows
 | Dependency | Version | License |
 |:-----------|:-------:|--------:|
-|actions/checkout|v2|MIT License|
 |actions/checkout|v3|MIT License|
 |actions/upload-artifact|v2|MIT License|
 |actions/upload-artifact|v3.1.0|MIT License|
 |aquasecurity/trivy-action|0.3.0|Apache License 2.0|
 |ASzc/change-string-case-action|v1|ISC License|
 |docker/build-push-action|v2|Apache License 2.0|
 |docker/login-action|v1|Apache License 2.0|
```

Comment:
Please increase version to v2, in v2 Node16 is introduced as default runtime

Reply:
Will be done in the next PR